protocol/avdl/chat1/local.avdl

```diff
@@ -1063,6 +1063,13 @@ protocol local {
   }
   GetTLFConversationsLocalRes getTLFConversationsLocal(string tlfName, TopicType topicType, ConversationMembersType membersType);
 
+  record GetChannelMembershipsLocalRes {
+    array<ChannelNameMention> channels;
+    boolean offline;
+    array<RateLimit> rateLimits;
+  }
+  GetChannelMembershipsLocalRes getChannelMembershipsLocal(keybase1.TeamID teamID, keybase1.UID uid);
```

Comment:
I'd change this to `gregor1.UID`, chat always takes that as the parameter here. JS can deal.